stage1/init/init.go

```diff
@@ -150,6 +152,19 @@ func machinedRegister() bool {
 	return found == 2
 }
 
+func copySystemd(p *Pod) error {
+	systemdBin := []string{
+		"/lib/systemd/systemd:/lib/systemd/systemd",
+		"/usr/bin/systemctl:/bin/systemctl",
```

Comment:
I would use `http://golang.org/pkg/os/exec/#LookPath` for getting paths to executable binaries from host. Figuring out `/lib/systemd/*` stuff might be hard to do reliably.